tests/test-cases/Group6-VIC-Machine/6-10-List.robot

```diff
@@ -54,6 +54,10 @@ Verify Listed Machines
     ${ret}=  Run  bin/vic-machine-linux inspect --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource ${vch-path} --name ${vch-name}
     Should Contain  ${ret}  Completed successfully
 
+Destroy Empty Cluster
```

Comment:
Can we remove the tests using this as we don't want to include anything that changes the CI VC config? We can add it to nightly test coverage.

Reply:
👍